src/plugins/gui/providers/ioniaProvider.ts

```diff
@@ -422,7 +423,8 @@ export const makeIoniaProvider: FiatProviderFactory<IoniaMethods> = {
         if (quoteParams.wallet == null) throw new Error('missing wallet')
 
         const rateAmount = await getCardPurchaseRateAmount(quoteParams.displayCurrencyCode, RATE_QUOTE_CARD_AMOUNT)
-        const rateExchangeAmount = await quoteParams.wallet.nativeToDenomination(toBigNumberString(rateAmount), quoteParams.displayCurrencyCode)
+        const multiplier = getCurrencyCodeMultiplier(quoteParams.wallet.currencyConfig, quoteParams.displayCurrencyCode)
+        const rateExchangeAmount = div(toBigNumberString(rateAmount), multiplier, multiplier.length)
```

Comment:
I don't think toBigNumberString is needed anymore now that biggystring has been updated; `.toString()` should be safe because all biggystring functions should accept scientific notation strings IIRC. It's worth a double take before removing `toBigNumberString` however.